research/src/pages/table.research.mdx

```diff
@@ -0,0 +1,19 @@
+---
+name: Table
```

Comment:
I noticed that Select is tagged as a working draft - should this be of the same status? Do we have a formalized process or enumerated criteria to inform what *state* this should be in?

Reply:
I'm not aware of the formalized process, but it's a good point, I'll mark Table as working draft as well